backend/app/users/models.py

```diff
@@ -10,6 +10,8 @@
 from sqlalchemy.ext.asyncio import AsyncSession
 from sqlalchemy.orm import Mapped, mapped_column
 
+from app.config import DEFAULT_API_QUOTA, DEFAULT_EXPERIMENTS_QUOTA
```

Comment:
Should be `..config import DEFAULT_API_QUOTA, DEFAULT_EXPERIMENTS_QUOTA` to avoid those mypy errors.
Also checking that you have run `pre-commit install` on your machine?